packages/pages/src/common/src/parsers/templateParser.ts

```diff
@@ -0,0 +1,32 @@
+import SourceFileParser, { createTsMorphProject } from "./sourceFileParser.js";
+
+/**
+ * TemplateParser is a class for parsing the template.
+ */
+export default class TemplateParser {
```

Comment:
This name is a little generic. We're doing something specific here. Can you name it accordingly?